enrollment-server/src/main/resources/db/changelog/changesets/20230302-add-column-fds-data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="enrollment-server-20230302-add-column-fds-data" author="racansky" labels="enrollment-server">
```

Comment:
Let's start to use liquibase from the very beginning of version 1.5.0 at least for dev and test environment (by default it turned off) and postgres only. Or should I introduce `context` to support both postgres and oracle?